moveit_core/distance_field/include/moveit/distance_field/propagation_distance_field.h

```diff
@@ -443,30 +440,32 @@ class PropagationDistanceField : public DistanceField
   {
     return max_distance_sq_;
   }
+  typedef std::vector<Eigen::Vector3i, Eigen::aligned_allocator<Eigen::Vector3i>> StlVectorEigenVector3i;
```

Comment:
When defining those typedefs, please follow the naming convention introduced by [EigenSTL](https://docs.ros.org/api/eigen_stl_containers/html/namespaceEigenSTL.html), i.e. `EigenSTL::vector_Vector3i`.